bentoml/yatai/yatai_service.py

```diff
@@ -123,6 +134,12 @@ def start_yatai_service_grpc_server(
                 '"pip install grpcio-reflection"'
             )
     server.add_insecure_port(f'[::]:{grpc_port}')
+    # TODO: it seems that current impl doesn't actually intercept RPC calls
```

Comment:
could you explain what does this TODO mean?

Reply:
if I didn't allocate a port for prometheus [beforehand](https://github.com/aarnphm/BentoML/blob/3175a0656c37af53963b81027e8acfe622245eea/bentoml/yatai/yatai_service.py#L138) seems that prometheus doesn't seem to register any metrics at all. This refers back to my question above is to whether we want to permanently assign `50052` to prometheus